src/extension.ts

```diff
@@ -578,6 +578,51 @@ export function activate(context: vscode.ExtensionContext) {
     )
   );
 
+  subscriptions.push(
+    vscode.commands.registerCommand(
+      "vscode-couchbase.getDocumentMetaData",
+      async (node: DocumentNode) => {
+        const connection = Memory.state.get<IConnection>("activeConnection");
+
+        if (!connection) {
+          return;
+        }
+        try {
+          const viewType = connection.url + "." + node.bucketName + "." + node.scopeName + node.collectionName + "." + node.documentName;
+          const result = await connection.cluster?.query(
+            `SELECT META(b).* FROM \`${node.bucketName}\`.\`${node.scopeName}\`.\`${node.collectionName}\` b WHERE META(b).id =  \"${node.documentName}\"`
+          );
+          if (currentPanel && currentPanel.viewType === viewType) {
+            currentPanel.webview.html = getDocumentMetaData(result.rows);
+            currentPanel.reveal(vscode.ViewColumn.One);
+          } else {
+            currentPanel = vscode.window.createWebviewPanel(
```

Comment:
Do we need a custom web view to view document metadata? Could we use a local json file like we do with document content?

Reply:
If we use a local json file, it will have many triggers (e.g. onDidSaveTextDocument) associated with it which we don't require. We need to separate it from normal json files so that we can differentiate.

Reply:
An alternative might be to use a different schema - eg we use the couchbase:// schema for document files, we could use metadata://, which would allows us to separate body from metadata.

Does the webview provide an editable view or read only?

Reply:
webview provide read-only. 
And I believe metadata we should be providing read only as it is on the web browser.

Reply:
Okay, let's go with web-view for now. We can replace with json doc if needed later.